src/cli/rdb-cli.c

```diff
@@ -372,9 +377,13 @@ int main(int argc, char **argv)
         return RDB_ERR_GENERAL;
     }
 
-    if ((logfile = fopen(options.logfilePath, "w")) == NULL) {
-        printf("Error opening log file for writing: %s \n", options.logfilePath);
-        return RDB_ERR_GENERAL;
+    if (strcmp(options.logfilePath, "-") == 0) {
```

Comment:
Should user pass `-l -` ? Is this a convention? If not, it seemed a bit error-prone to me. Maybe `-l stdout` is better (or something else without using `-`).

Reply:
As Oran mentioned below, using `stdout` might be an even less favorable option. I will discard this option.